src/auro-select.js

```diff
@@ -319,6 +323,35 @@ class AuroSelect extends LitElement {
     }
   }
 
+  /**
+   * Handles reading of auro-select by screenreaders.
+   * @private
+   * @returns {void}
+   */
+  labelForSr() {
+    const placeholderLabel = document.createElement("div");
+    const textId = "label";
+
+    placeholderLabel.setAttribute("id", textId);
+    placeholderLabel.setAttribute("aria-live", "polite");
+    placeholderLabel.classList.add("util_displayHiddenVisually");
+
+    this.addEventListener('focus', () => {
+      document.body.appendChild(placeholderLabel);
+
+      if (!this.optionSelected) {
+        document.getElementById(textId).innerHTML = this.placeholder;
+      } else {
+        // eslint-disable-next-line
+        document.getElementById(textId).innerHTML = `${this.optionSelected.innerText + ', ' + this.placeholder}`;
```

Comment:
Not sure why you did this, but wouldn't this work? 

```suggestion
       document.getElementById(textId).innerHTML = `${this.optionSelected.innerText}, ${this.placeholder}`;
```

Then we can remove the eslint ignore comment.

Reply:
Updated.